config/_ytt_lib/github.com/cloudfoundry/uaa/k8s/templates/values/version.yml

```diff
@@ -0,0 +1,3 @@
+#@data/values
+---
+version: "74.12.0"
```

Comment:
Side note: @shamus and @dbeneke we noticed a potential version mismatch while pulling in v74.15.0 . It looks like the value was added fairly recently but isn't automatically updated.

Reply:
Thanks! We've got a [story coming up](https://www.pivotaltracker.com/story/show/169720407) that should force us to fix this issue...we've also brought it up with our PMs to surface that it is currently causing pain.